spring-cloud-netflix-core/pom.xml

```diff
@@ -69,6 +69,10 @@
 			<artifactId>archaius-core</artifactId>
 			<optional>true</optional>
 		</dependency>
+		<dependency>
```

Comment:
Now this is a good question. If I understand the optional behaviour correctly, then if the user does not specify it (directly or indirectly) he wont have this dependency. If this is the case, then the current implementation would fail horribly due to a ClassNotFoundException in

```java
public boolean supports(Object object, Type bodyType, RequestTemplate template){
        return object instanceof Pageable;
    }
```
Since `Pageable` wont be available.


So would it require a ugly check like the one below, before using a class file which has an import to the optional dependency?
```java
try {
 Class.forName( "org.springframework.data.domain.Pageable" );
} catch( ClassNotFoundException e ) {
 // not here
}
```
If so, is this really desired?